### DISCUSSION TITLE
Vue 3 - Options API

### DISCUSSION THREAD

**Discussion by kevinmelo** [dm4t2/vue-currency-input#329] [Q&A] [answered]
This lib only works with `composition api`? or have support for the **vue 3** `options api`?

**Comment by dm4t2** [comment, marked as answer]
Options API might be supported in next major release because this requires to drop Vue 2 Support. Currently thinking about this...

**Reply by kevinmelo** [reply]
Got it, is not better to work only with directives?

**Reply by IAmShafqatAli** [reply]
Any Update regarding Options API support?
